scene/gui/code_edit.cpp

```diff
@@ -1007,6 +1007,116 @@ void CodeEdit::unindent_lines() {
 	queue_redraw();
 }
 
+void CodeEdit::convert_indent(int p_from_line, int p_to_line) {
+	if (!is_editable()) {
+		return;
+	}
+
+	// Check line range.
+	p_from_line = (p_to_line < 0) ? 0 : p_to_line;
```

Comment:
`p_from_line = (p_to_line < 0) ? 0 : p_to_line;`

Is this supposed to be `p_from_line`?

Reply:
Nice spot